cmake/Modules/OpmLibMain.cmake

```diff
@@ -218,10 +218,12 @@ endif (COMMAND install_hook)
 opm_install (${project})
 message (STATUS "This build defaults to installing in ${CMAKE_INSTALL_PREFIX}")
 
-# installation of CMake modules to help user programs locate the library
-include (OpmProject)
-opm_cmake_config (${project})
-
+if (NOT ${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+    #installation of CMake modules to help user programs locate the library
+    include (OpmProject)
+    opm_cmake_config (${project})
```

Comment:
At the moment, we do not need config mode, so we are fine with disabling this feature. The configuration issues for config mode on Windows was hard for me to understand, so I need someone else to fix these issues. Who can help with that task?